content/en/logs/error_tracking/browser_and_mobile.md

```diff
@@ -44,26 +43,22 @@ If you have not setup the Datadog Browser SDK yet, follow the [in-app setup inst
    })
    ```
 
-4. You must add the following snippet near initialization to catch and submit all uncaught exceptions to Datadog:
+3. To log a caught exception yourself, you may use [the optional error parameter][4]:
 
    ```javascript
-   window.onerror = function (message, source, lineno, colno, error) {
-       datadogLogs.logger.error(error?.message || '', {
-           error: { stack: error?.stack },
-       });
-   };
+   try {
+     throw new Error('wrong behavior');
+   } catch(err) {
+     datadogLogs.logger.error("an error occurred", {usr: {id: 123}}, err);
+   }
    ```
-5. To log a caught exception yourself, you may optionally use:
 
-   ```javascript
-   const e = new Error('an exception occurred');
-   datadogLogs.logger.error(e.message, {'error': {'stack': e.stack}});
-   ```
+**Note**: Error Tracking only considers errors that are instances of `Error`.
 
 [1]: https://app.datadoghq.com/logs/onboarding/client
 [2]: /logs/log_collection/javascript/#setup
-[3]: https://github.com/DataDog/browser-sdk/tree/main/packages/logs
-[4]: /logs/log_collection/javascript/#choose-the-right-installation-method
+[3]: /logs/log_collection/javascript/#choose-the-right-installation-method
+[4]: /logs/log_collection/javascript/?tab=npm#error-tracking
```

Comment:
Can you verify this link? It brought me to this page: https://docs.datadoghq.com/logs/log_collection/javascript/#npm, but I don't see a tab for npm or a header for #error-tracking

Reply:
I removed the wrong tab in https://github.com/DataDog/documentation/pull/17416/commits/ff0b4cd0f48e738f0ecfc7e2a54767e65c3c6d55 and the #error-tracking header should appear once https://github.com/DataDog/browser-sdk/pull/2098 is merged (I didn't know #documentation was pinged automatically, I should have put this in draft until this one was merged 😅 ).

Reply:
Pulled the latest master now that https://github.com/DataDog/browser-sdk/pull/2098 has been merged and the link is now redirecting to the right location.